launch_testing/launch_testing/markers.py

```diff
@@ -25,8 +26,13 @@ def keep_alive(test_description):
     return test_description
 
 
-def retry_on_failure(*, times):
-    """Mark a test case to be retried up to `times` on AssertionError."""
+def retry_on_failure(*, times, delay=0.1):
```

Comment:
Having `delay=0.1` changes the default behavior for this decorator, can you just use `0.0` or maybe `None` and do something like `if delay is not None: ...` instead? That makes approving this change easier for me.

Reply:
Done 8ed2390